src/views/figure_element.ts

```diff
@@ -142,6 +151,7 @@ export class FigureElementView extends BlockView<Trackable<FigureElementNode>> {
         files: groupFiles(doc, files),
         onUpload: handleUpload,
         onAdd: handleAdd,
+        onAddNewFile: handleNewFile,
       }
       this.reactTools = ReactSubView(
```

Comment:
move `this.reactTools?.remove()` to line 155

Reply:
done